src/tools/wasm-interp.cc

```diff
@@ -132,6 +208,39 @@ static void ParseOptions(int argc, char** argv) {
   parser.Parse(argc, argv);
 }
 
+Result RunSpecificExports(const Instance::Ptr& instance,
+                          Errors* errors,
+                          std::vector<FunctionCall>& calls) {
+  Result result = Result::Ok;
+
+  auto module = s_store.UnsafeGet<Module>(instance->module());
+  auto&& module_desc = module->desc();
+
+  for (auto&& export_ : module_desc.exports) {
+    if (export_.type.type->kind != ExternalKind::Func) {
+      continue;
```

Comment:
Should this be a fatal error?

Reply:
Not sure, that would make sense as well but I've decided to follow the already existent pattern https://github.com/WebAssembly/wabt/blob/494d145ec5425e71e2520d5b0427aacd6d2744ad/src/tools/wasm-interp.cc#L251

Reply:
hmm, but this case we are explicitly declaring we want to run an external function by name. Isn't like is iterating... Think fatal would make more sense, should I update @sbc100 ?

Reply:
I think the difference is that "--run-all-exports" really means "--run-all-exported-functions" so in that case it makes sense to skip things that a non-functions.  Bug if you explicitly say `--run-export=foo` I would expect that to fail if `foo` is not a function.

Reply:
I agree, updated the code and added an example. Please let me know anything